src/user-guide/writing-workflows/scheduling.rst

```diff
@@ -2015,22 +2015,25 @@ tools to limit workflow activity.
 Runahead Limiting
 ^^^^^^^^^^^^^^^^^
 
-The scheduler runahead limit determines how many consecutive cycle points can
-be active at once. The base point of the runahead calculation is the current
-lowest-valued point with :term:`active` or :term:`incomplete` tasks present.
+The runahead limit prevents a workflow from getting too far ahead of the oldest
+active cycle point.
 
-The runahead limit can be set with :cylc:conf:`[scheduling]runahead limit`,
-to an integer interval (for datetime or integer cycling workflows) or to
-a datetime interval (datetime cycling only).
+A cycle is considered to be active if it contains any :term:`active` or
+:term:`incomplete` tasks.
+
+The runahead limit can be configured for a workflow by
+:cylc:conf:`[scheduling]runahead limit`. It defines the span of cycles which
+are permitted to run as measured from the oldest active cycle point. The
+runahead limit can be defined in two ways, as an integer or datetime span.
+
+.. rubric:: Integer Format
 
 The integer format ``Pn`` is an interval that spans ``n+1`` consecutive cycle
 points, regardless of how the cycle point values increment from one point to
 the next (i.e. regardless of the cycling interval).
 
 For example, the default runahead limit of ``P4`` allows 5 active cycle points
-(base point plus the 4 next points) whether the cycling is defined by
-``R/^/P1`` or ``R/^/P3`` etc. (for integer cycling) or ``R/^/PT6H`` etc. (for
-datetime cycling).
+(the oldest active cycle plus the 4 next cycles).
 
 In the following example, tasks ``1/foo`` through ``4/foo`` will submit to
 run immediately at start-up, but ``5/foo`` will be held back until ``1/foo``
```

Comment:
No! The example uses P2 at the moment. Either correct these or change the example to use P1?